static/src/javascripts/projects/common/modules/identity/consent-journey.js

```diff
@@ -16,10 +16,10 @@ const showJourney = (journeyEl: HTMLElement): Promise<void> =>
 const hideLoading = (loadingEl: HTMLElement): Promise<void> =>
     fastdom.write(() => loadingEl.remove());
 
-const shouldDisplaySectionWarning = (
+const uncheckedBoxes = (
     journeyEl: HTMLElement,
     section: string
-): Promise<boolean> =>
+): Promise<Object> =>
```

Comment:
do you need `uncheckedBoxes().total` anymore? This could be simplified to not have a wrapper object